bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/RuleStatusInfo.java

```diff
@@ -69,6 +74,7 @@ public RuleStatusInfo(RuleStatus status, RuleStatusDetail statusDetail, String d
         this.status = status;
         this.statusDetail = statusDetail;
         this.description = description;
+        this.properties = properties;
```

Comment:
I will need a deeper look into this, but I am not convinced that such properties should be introduced here (as it brings it out of sync with the Thing status concept and also most likely breaks all our evening mechanism...).